molgenis/molgenis-emx2
# docs: changed styling for general documentation

What are the main changes you did:
- Changed the styling for the documentation

how to test:
- Check out the branch
- http://localhost:8080/apps/docs/#/
- See the difference

todo:
